handlers/create.go

```diff
@@ -125,18 +129,24 @@ func logProgress(readerStdout io.Reader, readerStderr io.Reader, publishChan cha
 	}
 }
 
-func filterDockerMessage(msg string, uuid string, errString *string) string {
+func filterDockerMessage(msg string, machine *client.Machine, errString *string) string {
```

Comment:
I think this function should have a unit test that passes in some messages and asserts that the oiutput is as expected.